examples/ccl_sample_angpow.cpp

```diff
@@ -0,0 +1,198 @@
+#include <stdlib.h>
```

Comment:
Question: why is this a .cpp file? Angpow is C++ but I thought CCL is still a C library, so you could link it from any C code. Is this no longer true?

Also, I thought we were using .cc and not .cpp for C++ source files

Reply:
You're right there is not point that this is a C++ file. I moved it to C only.
The files in examples use the extension .cpp, so I did same to preserve the harmony of CCL ;)